src/pages/TransactionHistory/TransactionHistoryFilter.tsx

```diff
@@ -98,7 +101,7 @@ export const TransactionHistoryFilter = memo(
     const RangeCustomComponent = useCallback(() => {
       return dayRangeSelectedOption === customRangeOption ? (
         <HStack px={4} my={2} alignItems='center' mx={-4}>
-          <DatePicker name={FilterFormFields.FromDate} control={control} />
+          <DatePicker name={FilterFormFields.FromDate} control={control} withPortal />
```

Comment:
Needs to be diffed in l.111 too

Reply:
Fuck me... 

<img width="699" alt="image" src="https://github.com/user-attachments/assets/4be1f0be-cf5a-46a3-a16b-4eb67d65bac2" />

Added in [6514fc6](https://github.com/shapeshift/web/pull/9082/commits/6514fc6ada3959c4318a6d5ff417208aa8853d6e)